src/pages/api/auth/create.ts

```diff
@@ -82,6 +84,27 @@ async function handler(req: NextApiReq, res: NextApiRes) {
 
   delete newUser.password;
 
+  if (limits) {
+    const data = {};
+    const ends = ['type_time', 'limit_by', 'limit'];
+    for (const [x, y] of Object.entries(limits)) {
+      if (ends.includes(x)) data[x] = [y][0];
+      else continue;
+    }
```

Comment:
```suggestion
    for (const [x, y] of Object.entries(limits)) {
      if (ends.includes(x)) data[x] = [y][0];
    }
```
Useless jump statement. It'll continue forward regardless of whether or not the `if` check passes.